test/integration/features/basic.feature

```diff
@@ -28,4 +36,4 @@ Feature: Basic
      When executing "minishift stop"
      Then Minishift should have state "Stopped"
      When executing "minishift delete"
-     Then Minishift should have state "Does Not Exist"
+     Then Minishift should have state "Does Not Exist"
```

Comment:
Check...

but we can get the IP of the VM. this is `minishift ip`, and we can actually store this if needed... we will probably need this more often. But let's leave this as a possible improvement.

We could collect a list for this in a new issue.